apluslms_roman/configuration.py

```diff
@@ -20,29 +22,31 @@
     ),),)),
 ))
 
-
 class CourseConfigError(Exception):
     pass
 
-
 def is_legacy_configuration(path):
-    """Checks if path is legacy a-plus-rst-tools course"""
+    # Checks if path is legacy a-plus-rst-tools course
```

Comment:
Why is the doc string changed to a comment? 🤔

Reply:
I thought it was unnecessary since it's a one-line comment, is there some other reason to use strings?

Reply:
okay googled it, changing it back